docs/docsite/rst/roadmap/COLLECTIONS_8.rst

```diff
@@ -49,7 +49,7 @@ Ansible 8.x minor releases will occur approximately every four weeks if changes
 
 .. note::
 
-    Minor releases will stop when Ansible-8 is released.  See the :ref:`Release and Maintenance Page <release_and_maintenance>` for more information.
+    Minor releases will stop when Ansible-9 is released.  See the :ref:`Release and Maintenance Page <release_and_maintenance>` for more information.
```

Comment:
@felixfontein AFAIK we won't release any bug fixes (rare as they are) either, so maybe we should mention this explicitly: Both minor and bugfix releases will stop when Ansible-9 is released.

Or just say that  there will be no more Ansible-8.x releases after Ansible-9 is released.